views/browse.slim

```diff
@@ -54,8 +54,8 @@ html lang="en"
               <path fill-rule="evenodd" clip-rule="evenodd" d="M5.77778 0C8.9688 0 11.5556 2.5868 11.5556 5.77778C11.5556 7.21324 11.032 8.52649 10.1655 9.53698L10.4063 9.77778H11.1111L15.5556 14.2222L14.2222 15.5556L9.77778 11.1111V10.4063L9.53698 10.1655C8.52649 11.032 7.21324 11.5556 5.77778 11.5556C2.5868 11.5556 0 8.9688 0 5.77778C0 2.5868 2.5868 0 5.77778 0ZM5.77778 1.77779C3.56864 1.77779 1.77778 3.56865 1.77778 5.77779C1.77778 7.98693 3.56864 9.77778 5.77778 9.77778C7.98693 9.77778 9.77778 7.98693 9.77778 5.77779C9.77778 3.56865 7.98693 1.77779 5.77778 1.77779Z" fill="white"/>
             </svg>
 
-    div.datastores-nav
-      nav class="viewport-container menu-nav" aria-label="datastores"
+    div class="datastores-nav"
+      nav class="viewport-container menu-nav" aria-label="Datastores navigation"
```

Comment:
It's not necessary to include the role in a label. It would read kind of repetitive, eg, "Datatores navigation, navigation." It's kind of like if we wrote `<button>Search button</button>`.

```suggestion
      nav class="viewport-container menu-nav" aria-label="Datastores"
```